psk_fitness/psk_fitness/Interfaces/IExerciseRepository.cs

```diff
@@ -5,11 +5,13 @@ namespace psk_fitness.Interfaces
 {
     public interface IExerciseRepository
     {
+        Task<List<ExerciseForWorkoutDTO>> GetExercisesForWorkout(string userEmail);
```

Comment:
The naming is weird. It should use both userEmail or userId AND WorkoutId

Reply:
Why do you need THE workoutId when a user can choose from his created exercises? But the function name could be longer and more informative I'll take care of it

Reply:
Okay, so I understand it should be GetAvailableExercises. They are not tied to a Workout here.